dm/master/server.go

```diff
@@ -217,7 +217,11 @@ func (s *Server) recoverSubTask() error {
 		return err
 	}
 	for _, kv := range resp.Kvs {
-		infos := common.UpstreamSubTaskKeyAdapter.Decode(string(kv.Key))
+		infos, err := common.UpstreamSubTaskKeyAdapter.Decode(string(kv.Key))
+		if err != nil {
+			log.L().Warn("decode upstream subtask key from etcd failed", zap.Error(err))
```

Comment:
how about adding the key 's information in log

Reply:
The key's information has be contained in error returned from function **Decode**.